Dockerfile

```diff
@@ -1,8 +1,6 @@
 FROM docker/compose:alpine-1.29.2
 
 COPY entrypoint.sh /entrypoint.sh
-COPY validate /github-actions-validate
-
-RUN ["chmod", "+x" ,"/entrypoint.sh"]
+COPY validate /assert-tables
```

Comment:
path is `validate` and not `test`.

Reply:
I'm guessing this rename needs to happen in many places, so I'll stop commenting on each.

Reply:
I think `assert-tables` is a good directory name, since that is in line with the action